### DISCUSSION TITLE
Screen brightness adjustment in dock

### DISCUSSION THREAD

**Discussion by lxwndr** [linuxdeepin/developer-center#3253] [Features Request & Ideas | 特性请求 & 头脑风暴]
Check the possibility of adding a bar or an icon in the dock to adjust the brightness of the screen.

By placing the cursor over the location, rotating the scroll can quickly adjust the intensity of the screen's brightness - similarly, the volume icon.

deepin Community 20.6 (OsBuild=11038.004)

**Comment by zccrs** [comment]
Good
